tests/runtimes/android/android_runtime_tests.py

```diff
@@ -478,3 +478,25 @@ def test_446_test_foreground_not_sticky_services_are_working(self):
         assert service_name not in service_info, "{0} service found! Logs: {1}".format(service_name, service_info)
         assert pid not in service_info, "{0} service with id {1} found! Logs: {2}".format(service_name, pid,
                                                                                           service_info)
+
+    def test_447_test_useV8Symbols_flag_in_package_json(self):
+        """
+         https://github.com/NativeScript/android-runtime/issues/1368
+        """
+        Folder.clean(os.path.join(TEST_RUN_HOME, APP_NAME))
+        Tns.create(app_name=APP_NAME, template=Template.HELLO_WORLD_JS.local_package, update=True)
+
+        Tns.platform_add_android(APP_NAME, framework_path=Android.FRAMEWORK_PATH)
+        log = Tns.build_android(os.path.join(TEST_RUN_HOME, APP_NAME), verify=False)
```

Comment:
New app with brand new platform will force full build, do we really need it?

Reply:
This was for testing purpose will be removed soon.

Reply:
Removed as I said.